components/dialogs/BaseDialog.vue

```diff
@@ -0,0 +1,175 @@
+<template>
+  <v-dialog
+    v-model="isDialogOpen"
+    :max-width="width"
+    v-bind="getDialogProps"
+    content-class="base-dialog--likecoin"
+  >
+
+    <header class="base-dialog--likecoin__header">
```

Comment:
For BEM naming, modifier is after element e.g.
`base-dialog--likecoin__header` → `base-dialog__header`

So you can use
```scss
.base-dialog__header {
  // For global properties

  .base-dialog--likecoin {
     // For modifier specific properties
  }
}
```
...and the classes below